codingStandards/AdditionalGuidelines-Java.md

```diff
@@ -0,0 +1,44 @@
+# Additional Guidelines - Java
+
+## Using assertions
+
+Refer the article 
+_[Programming With Assertions](http://docs.oracle.com/javase/7/docs/technotes/guides/language/assert.html)_ 
+(from Oracle) for more details on the three general guidelines below.
```

Comment:
- "Refer `to` the article"
- Should we be linking [.../javase/`8`/docs/...](http://docs.oracle.com/javase/8/docs/technotes/guides/language/assert.html) instead?